packages/test-utils/sdks/alby/tsconfig.json

```diff
@@ -1,7 +1,10 @@
 {
   "compilerOptions": {
-    "module": "NodeNext",
-    "resolveJsonModule": true
+    "esModuleInterop": true,
+    "module": "ESNext",
+    "moduleResolution": "Bundler",
+    "resolveJsonModule": true,
+    "strict": true
```

Comment:
is `attw` still happy with these changes?

Reply:
good reminder, i hadn't checked that — but yes! 

![CleanShot 2023-10-25 at 13 55 04@2x](https://github.com/readmeio/api/assets/8854718/59b8674d-40dc-44e5-bc1d-9bb5ada01732)